### PR TITLE
Fix isAlive always true bug

### PR DESCRIPTION
**Pull request by liayoo**
- Fixed isAlive always being true
- Updated endpointUrls & checkin/checkoutIds in tools configs
- Updated start_node_genesis_gcp.sh to set PEER_CANDIDATE_JSON_RPC_URL to node 0's address only for nodes 0-9

### CODE REVIEW

**Comment by minhoryang** [comment]
Why don't we just swap the ordering of 'Object.assign()' argument?

**Comment by liayoo** [comment]
> Why don't we just swap the ordering of 'Object.assign()' argument?

Thanks for asking! That would totally work, but since only the isAlive value needs changing, it just seemed simpler~

File: start_node_genesis_gcp.sh [hunk 1]
```diff
@@ -168,7 +168,10 @@ elif [[ $SEASON = 'summer' ]]; then
     KEYSTORE_DIR=testnet_prod_keys
 elif [[ "$SEASON" = "sandbox" ]]; then
     export BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/testnet-sandbox
-    export PEER_CANDIDATE_JSON_RPC_URL="http://130.211.244.169:8080/json-rpc"
+    if [[ $NODE_INDEX -lt 10 ]]; then
```
Comment (platfowner) [inline start_node_genesis_gcp.sh:171]: 👍

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Comment by liayoo** [comment]
Thanks for the reviews! Now merging~

**Pull request merged**